modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
+
+Operators can manage other Operators within their component. This lends to a
+transparent model: a single Operator usually manages a single binary file.
+Operators also offer a more granual configuration experience. You configure each component by
+modifying its Operator instead of modifying a global configuration file.
```

Comment:
Modifying the API an operator exposes instead of modifying a global configuration file